docs/TektonConfig.md

```diff
@@ -602,17 +602,18 @@ The following fields are supported in `deployment`
       * `nodeSelector` - replaces the existing NodeSelector with this, if not empty
       * `tolerations` - replaces the existing tolerations with this, if not empty
       * `topologySpreadConstraints` - replaces the existing TopologySpreadConstraints with this, if not empty
+      * `runtimeClassName` - adds and updates runtimeClassName
```

Comment:
update the same in `StatefulSets` documentation too

Reply:
Done, thanks 👍🏻